src/documents/Priorities.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Priorities
+layout: default
+category: API
```

Comment:
I'm not convinced this page should be in the API category. It also needs to be added to the Table of Contents, in whichever category we end up putting it in.